### PR TITLE
Avoid path too long errors when performing BuildV2 builds in MVC repo

### PR DESCRIPTION
**Pull request by dougbu**
- do not glob to the ends of the earth when looking for project.json files

nits:
- remove unused variable in `_kpm-build.shade`
- remove incorrect comment in `_kpm-pack.shade`
- clean up some trailing whitespace

### CODE REVIEW

File: build/_k-standard-goals.shade [hunk 1]
```diff
@@ -115,31 +115,35 @@ default NUGET_FEED = 'https://api.nuget.org/v3/index.json'
 #build-compile target='compile' if='IsBuildV2'
   @{
       // If the src folder, build and create the packages
-      if (Directory.Exists("src")) 
+      if (Directory.Exists("src"))
       {
-          DnuPack("src/**", BUILD_DIR, Configuration);
-          
+          DnuPack("src/*;src/*/*;src/*/*/*", BUILD_DIR, Configuration);
```
Comment (victorhurdugaci) [inline build/_k-standard-goals.shade]: Please add a comment to explain why we do this

**Comment by victorhurdugaci** [comment]
looks good :shipit:

File: build/_k-standard-goals.shade (hunk 1 above)
Comment (dougbu) [inline build/_k-standard-goals.shade]: done

**Comment by dougbu** [comment]
cb3dea6

**Pull request closed without merging**